bench/workgen/workgen.cpp

```diff
@@ -105,6 +105,8 @@ extern "C" {
 #define OP_HAS_VALUE(op) \
     ((op)->_optype == Operation::OP_INSERT || (op)->_optype == Operation::OP_UPDATE)
 
+#define DYN_TABLE_APP_METADATA "app_metadata=\"dynamic_table\""
```

Comment:
Could we add/change the value to specify "workgen"? Either `workgen_table` or `workgen_dynamic_table`, perhaps? There could be future potential value in knowing the tool that created the table.

Reply:
Fixed.